vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_footer.scss

```
@@ -0,0 +1,91 @@
+footer {
+	background: $grey;
+	color: $white;
+	position: relative;
+	text-align: center;
+	padding-top: 5rem ;
+	padding-bottom: 10px;
+  overflow: hidden;
```

@nykka this line is not align with the others

```suggestion
	overflow: hidden;
```